config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root 'users#new'
+  resources :users, only: %i[create new edit]
```

Comment:
Are these all the methods that you need? Did you compare these methods with the ones in `users_controller.rb`? Have you tried to edit an user by accessing `/users/1/edit`? Please create a new user and try it accessing the the page.

[`REQUIRED`]